recipes/perfetto/all/conandata.yml

```diff
@@ -1,7 +1,4 @@
 sources:
-  "20.1":
-    url: https://github.com/google/perfetto/archive/refs/tags/v20.1.tar.gz
-    sha256: d681bb76e2b73e6ba46db53c1502f31f4f16c36cd6e91d4ae839a3b44272f646
```

Comment:
why removing 20.1 ? is it because it has some failure when disable_logging is True ? In this case please raise an exception in validate(self)

Reply:
Nope... it's just too old and requires one more separate patch folder to support the setting. Do we wanna support it still?

Reply:
20.1 is back with it's own patch now